CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,157 @@
+# Zarf Contributor Ladder
+
+This document outlines the various contributor roles in the Zarf project, along with their respective prerequisites and responsibilities.
+It also defines the process by which users can request to change roles.  These roles are progressive, in that responsibilites and pre-requisites apply to subsequent rungs in the ladder.
+
+- [Roles](#roles)
+  - [Community Participants](#community-participants)
+  - [Contributors](#contributors)
+  - [Reviewers](#reviewers)
+  - [Maintainers](#maintainers)
+- [Inactive members](#inactive-members)
+
+## Roles
+
+### Community participants
+
+Community participants engage with Zarf, 
+contributing their time and energy in discussions or just generally helping out.  Additionally, community members participate in [Zarf community meetings](https://github.com/zarf-dev/zarf/issues/2613).
+
+#### Responsibilities
+
+- Keep participating!
+
+#### Prerequisites
+
+- Must follow the [OpenSSF Code of Conduct]
+- Must follow the [Contribution Guide]
+
+### Contributors
+
+Contributors help advance the Zarf project through commits, issues, and pull requests.  Contributors participate through GitHub teams,
```

Comment:
I don't believe we would manage an actual "role" for this level in the organization - so the automation of test workflows may be a trait of the reviewers and above role?